app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    short_command, name_f_in, name_f_out = command.split()
```

Comment:
great, but use full names for ur variables

Reply:
corrected